docs/tutorials/getting_started.rst

```diff
@@ -0,0 +1,290 @@
+your specific backend type for valid options.
+
+Transpile options
+-----------------
+These options are passed to the Terra transpiler to transpile the experiment circuits
+before execution:
+
+.. code-block::
+
+  exp.set_transpile_options(scheduling_method='asap',
+                            optimization_level=3,
+                            basis_gates=["x", "sx", "rz"])
+
+Consult the documentation of :func:`qiskit.compiler.transpile` for valid options.
+
+Experiment options
+------------------
+These options are unique to each experiment class. Many experiment options can be set
+upon experiment instantiation, but can also be explicitly set via 
+:meth:`~BaseExperiment.set_experiment_options`:
+
+.. code-block:: 
+
+  exp = T1(physical_qubits=(i,), delays=delays)
+  exp.set_experiment_options(delays=new_delays)
+
+Consult the :doc:`API documentation </apidocs/index>` for the options of each experiment class.
+
+Analysis options
+----------------
+
+These options are unique to each analysis class. Unlike the other options, analyis options 
+are not directly set via the experiment object
+but use instead a method of the associated ``analysis``:
+
+.. code-block::
+
+  exp = rb.StandardRB(physical_qubits=(0,),
+                      lengths=list(range(1, 300, 30)),
+                      seed=123,
+                      backend=backend)
+  exp.analysis.set_options(gate_error_ratio=None)
+
+Consult the :doc:`API documentation </apidocs/index>` for the options of each 
+experiment's analysis class.
+
+Running experiments on multiple qubits
+======================================
+
+To run experiments across many qubits of the same device, we use **composite experiments**.
+A composite experiment is a parent object that contains one or more child 
+experiments, which may themselves be composite. There are two core types of composite experiments:
+
+* **Parallel experiments** run across qubits simultaneously as set by the user. New circuits 
+  are constructed such that parallelize circuits of the child experiments are executed 
+  simultaneously. Therefore, the circuits in child experiments *cannot* overlap in qubits 
+  used. The marginalization of measurement data for analysis of each child experiment is handled automatically. 
+* **Batch experiments** run consecutively in time. These child circuits *can* overlap in qubits used.
+
+Here's an example of measuring the :math:`T_1` of two qubits on the same device simultaneously
+in a parallel experiment:
+
+.. jupyter-execute::
+
+    from qiskit_experiments.framework import ParallelExperiment
+
+    parallel_exp = ParallelExperiment([T1(physical_qubits=(i,), delays=delays) for i in range(2)])
+    parallel_data = parallel_exp.run(backend, seed_simulator=101).block_for_results()
+
+Note that when options are set for a composite experiment, the child 
+experiments's options are also set recursively. Let's examine how the parallel 
```

Comment:
I don't think it's always true. It depends on the option.

Reply:
Good catch, updated